### PR TITLE
13021 - Part 2 - Update storybook files.

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:* /bcgov/entity#13021

*Description of changes:*
Update storybook files

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — COMMENTED** [review]
Were you using Node v12 to generate these?

There should only be a handful of diffs here.

**Comment by seeker25** [comment]
Yes node v12.22.22

**Comment by seeker25** [comment]
Let me try again

**Comment by seeker25** [comment]
No changes..

```
nvm use 12.22.12
npm i 
npm i --global lerna

trav@STOMP-COMP:~/work/bcrs-shared-components$ lerna bootstrap --hoist
info cli using local version of lerna
lerna notice cli v3.22.1
lerna info versioning independent
lerna info Bootstrapping 25 packages
lerna info Installing external dependencies
lerna info hoist Installing hoisted dependencies into root
lerna info hoist Pruning hoisted dependencies
lerna info hoist Finished pruning hoisted dependencies
lerna info hoist Finished bootstrapping root
lerna info Symlinking packages and binaries
lerna success Bootstrapped 25 packages
trav@STOMP-COMP:~/work/bcrs-shared-components$ npm run storybook:build
trav@STOMP-COMP:~/work/bcrs-shared-components$ npm run storybook
```

**Comment by severinbeauvais** [comment]
Weird. When I do it (using node v 12.22.8), I only see 3 diffs:

image

**Comment by severinbeauvais** [comment]
Same when I use node 12.22.12:

image

**Comment by severinbeauvais** [comment]
Let's close this PR and please review this one instead: https://github.com/bcgov/bcrs-shared-components/pull/147

We can figure this out later. I'm swamped today.
